core/vocabulary/src/test/java/org/trellisldp/vocabulary/AbstractVocabularyTest.java

```diff
@@ -99,16 +99,13 @@ public void testVocabularyRev() {
 
     @Test
     public void testNamespace() throws Exception {
-        final Optional<Field> uri = stream(vocabulary().getFields()).filter(field -> field.getName().equals("URI"))
-                .findFirst();
-
-        assertTrue(uri.isPresent(), vocabulary().getName() + " does not contain a 'URI' field!");
-        assertEquals(namespace(), uri.get().get(null), "Namespaces do not match!");
+        final Method m = vocabulary().getMethod("getNamespace");
+        assertEquals(namespace(), m.invoke(null), "Namespaces do not match!");
     }
 
     private Stream<String> fields() {
         return stream(vocabulary().getFields()).map(Field::getName).map(name ->
                 name.endsWith("_") ? name.substring(0, name.length() - 1) : name)
-            .map(name -> name.replaceAll("_", "-")).filter(field -> !field.equals("URI"));
+            .map(name -> name.replaceAll("_", "-")).filter(field -> !field.equals("NS"));
```

Comment:
The `Class::getFields` method will return all of the public fields of the class, so there is no more need to filter by `field -> !field.equals("NS")`; in fact, that filter should be entirely removed, since "NS" may be used at some point as a public field.